configure.ac

```diff
@@ -225,3 +230,32 @@ AC_ARG_WITH([cc-image],
 )
 
 AC_OUTPUT
+
+dnl === Summary ===============================================================
+
+AS_IF([test x"$disable_tests" = xno], [enable_tests=yes], [enable_tests=no])
+AS_IF([test x"$disable_functional_test" = xno],
+      [enable_functional_tests=yes], [enable_functional_tests=no])
+
+echo ""
```

Comment:
Rather than raw `echo` calls, I'd use `AC_MSG_RESULT`.

For example, https://github.com/jamesodhunt/procenv/blob/master/configure.ac#L212.